tokenizer.py

```diff
@@ -356,6 +357,17 @@ def t_OPMUL(t):
     t.lexer.suffixesSeen = set()
     return t
 
+# A multiplication long
+@lex.TOKEN(r'(' + r'(?=[A-Z\t ])|'.join([k for k,v in instrInfos.exportInstrInfo.items() if v == instrInfos.InstrType.multiplylong])+r'(?=[A-Z\t ]))')
+def t_OPMULL(t):
+    t.lexer.begin('mulopcode')
+    t.lexer.currentMnemonic = t.value
+    t.lexer.countArgs = 0
+    t.lexer.instrType = instrInfos.InstrType.multiply
+    t.lexer.expectedArgs = 3
```

Comment:
expectedArgs should be at 4, right? But I admit this value is not used at this time (I just added it in case it would be helpful for raising better error messages)